website/blog/2022-01-24-docusaurus-2021-recap/index.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Docusaurus 2021 Recap
+authors: [Josh-Cena]
+tags: [recap]
+image: ./img/thumbnail.png
+---
+
+Docusaurus has made great progress in 2021! We've **reached full feature parity** with v1 after [the release of internationalization](../2021-03-09-releasing-docusaurus-i18n.md), [announced our **first beta**](../2021-05-12-announcing-docusaurus-two-beta/index.md), and welcomed **significantly more users** this year. We also reached our **30k-star milestone** at the start of 2022!
+
+The **official v2 release** is just around the corner! Follow the [roadmap issue](https://github.com/facebook/docusaurus/issues/6113) for latest news.
+
```

Comment:
what about adding the social image here too?

I find it nice to have an image on the list view, like we did in a few posts

![Uploading image.png…]()

Reply:
Of course 👍